Src/java/cql-to-elm/src/test/resources/org/cqframework/cql/cql2elm/fhir/r401/TestOverloadForward.cql

```diff
@@ -0,0 +1,18 @@
+library TestOverloadForward
+
+using FHIR version '4.0.1'
```

Comment:
Question: Do we need to consider the case of header-only declarations for an included library? I _think_ given the current compiler architecture it's expected that an included Library will be fully compiled but I'm not 100% certain.

Reply:
Yes, the expectation is that included libraries are fully compiled and the current translator will ensure that is the case. I tried to do lazy compilation even for included libraries, but it proved to be more trouble than it was worth at this point.